estate/views/estate_property_offer_views.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_offer_action" model="ir.actions.act_window">
```

Comment:
Actions are always at the end of the file due to the sequential nature of XML parsing. To prevent reference of "not yet" created views